src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,2105 @@
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <param name="exclusiveCommandOption">Exclusive command option</param>
+        /// <exception cref="InvalidOperationException">This exception can be due to an invalid state.</exception>
+        /// <exception cref="InvalidOperationException">This exception can be due to operation failed.</exception>
+        /// <exception cref="OutOfMemoryException">This exception can be due to out of memory.</exception>
+        /// <pre>
+        /// The State must be Ready.
+        /// </pre>
+        /// <post>
+        /// The Service State will become Recording.
+        /// </post>
+        public static void Start(bool exclusiveCommandOption)
+        {
+            ErrorCode error = VcMgrStart(exclusiveCommandOption);
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "Start Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+        }
+
+        /// <summary>
+        /// Stop recognition.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <exception cref="InvalidOperationException">This exception can be due to an invalid state.</exception>
+        /// <exception cref="InvalidOperationException">This exception can be due to operation failed.</exception>
+        /// <pre>
+        /// The State must be Recording.
+        /// </pre>
+        /// <post>
+        /// The Service State will become Processing.
+        /// </post>
+        public static void Stop()
+        {
+            ErrorCode error = VcMgrStop();
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "Stop Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+        }
+
+        /// <summary>
+        /// Cancels recognition.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <exception cref="InvalidOperationException">This exception can be due to an invalid state.</exception>
+        /// <exception cref="InvalidOperationException">This exception can be due to operation failed.</exception>
```

Comment:
If this exception can be thrown in both cases, please write it as a single instance of `<exception cref="InvalidOperationException">`.